src/utils/column-helper.ts

```diff
@@ -16,14 +16,15 @@ export function setColumnDefaults(columns: any[]) {
       column.$$id = id();
     }
 
+    // prop can be numeric; zero is valid not a missing prop
     // translate name => prop
-    if(!column.prop && column.name) {
+    if(column.prop == null && column.name) {
```

Comment:
If we're certain that `column.prop` is `number|undefined`, then that's fine. I just did it because the original `!column.prop` would have caught `null|undefined|0`, but with numeric 0 being valid, I wanted to catch `null|undefined`

Maybe `setColumnDefaults` should throw an error on a `null` `column.prop` to enforce the assumption?